packages/components/src/PlaceCard/Placeholder.js

```diff
@@ -0,0 +1,74 @@
+// @TODO see if it is still usable somehow: react-native-shimmer must be linked, so it will not work in Expo.
```

Comment:
What are your plans with this? 
I don't like to have not working code in the repo, because at some point we can easy forgotten about it, but if you have plans to look at it soon it's OK.